chain/ethereum/ethereum.go

```diff
@@ -35,6 +35,11 @@ type Connector struct {
 	Sub    ISubscription
 }
 
+const (
```

Comment:
is this something that can change per rpc/per connector? should we add it to config and have a default set to these values?

Reply:
That is mostly node-specific but around these values. Providers also don't change it often, or at all. 

We could add it to the config but since it is only used here, I think it can be hardcoded too. 

I wanted to make them global because different functions in that module are referring to them.